staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -145,12 +140,11 @@ func New(
 			// No need to handle deletion event because the deletion would be handled by
 			// the update path when the deletion timestamp is added.
 		},
-		serviceSyncPeriod,
 	)
 	s.serviceLister = serviceInformer.Lister()
 	s.serviceListerSynced = serviceInformer.Informer().HasSynced
 
-	nodeInformer.Informer().AddEventHandlerWithResyncPeriod(
+	nodeInformer.Informer().AddEventHandler(
```

Comment:
This one indeed seems correct - if none of the nodes changed, there doesn't seem to be a need to resync any of the existing LBs/services etc.

That said, please add a comment explaining why we don't need to periodically resync here - to ensure that someone else comes and readd that again in the future.